test/hotspot/jtreg/runtime/valhalla/inlinetypes/TestValue4.java

```diff
@@ -44,41 +44,27 @@
     final long l;
     final String val;
 
-    private TestValue4() {
-        i = (int)System.nanoTime();
-        val = Integer.valueOf(i).toString();
-        l = ((long)i) << Integer.SIZE | i;
-        s1 = (short)(i & ~Short.MIN_VALUE);
-        s2 = (short)(i >> Short.SIZE);
-        b1 = (byte)(i & ~Byte.MIN_VALUE);
-        b2 = (byte)((i >> Byte.SIZE) & ~Byte.MIN_VALUE);
-        b3 = (byte)((i >> (2 * Byte.SIZE)) & ~Byte.MIN_VALUE);
-        b4 = (byte)((i >> (3 * Byte.SIZE)) & ~Byte.MIN_VALUE);
+    public TestValue4() {
+        this((int) System.nanoTime());
```

Comment:
Don't need the bit twiddling logic, because it's reproduced by the ByteBuffer reads in the other constructor.

Reply:
At some point early on, debugging the interpreter was probably easy to not leave the frame, hence the manual bit twiddle, but I think we are past that now, nice clean up.